scriptworker/cot/verify.py

```diff
@@ -1215,10 +1215,13 @@ def verify_link_in_task_graph(chain, decision_link, task_link):
     ).get_commit(commit_hash)
 
     committer_login = commit_data['committer']['login']
+    committer_email = commit_data['commit']['committer']['email']
     # https://github.com/mozilla-releng/scriptworker/issues/334: web-flow is the User used by
     # GitHub to create some commits on github.com or the Github Desktop app. For sure, this user
     # is not the one who triggered a push event. Let's fall back to the author login, instead.
-    sender_login = commit_data['author']['login'] if committer_login == 'web-flow' else committer_login
+    if committer_login == 'web-flow':
+        committer_login = commit_data['author']['login']
+        committer_email = commit_data['commit']['author']['email']
```

Comment:
To be perfectly honest, I inferred that line. I haven't found a way to test it out. I believe it requires several components including using the Github Desktop app.